datasets/cms_medicare/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import fnmatch
+import json
+import logging
+import os
+import pathlib
+import typing
+from zipfile import ZipFile
+
+import pandas as pd
+import requests
+from google.cloud import storage
+
+
+def main(
+    source_url: str,
+    source_file: pathlib.Path,
+    target_file: pathlib.Path,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+    headers: typing.List[str],
+    rename_mappings: dict,
+    pipeline_name: str,
+) -> None:
+
+    logging.info(
+        "CMS Medicare process started at "
+        + str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+    )
+
+    logging.info("creating 'files' folder")
+    pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
+
+    logging.info(f"Downloading file {source_url}")
+    download_file(source_url, source_file)
+
+    logging.info(f"Opening file {source_file}")
+
+    if (
+        pipeline_name == "inpatient_charges_2011"
+        or pipeline_name == "inpatient_charges_2012"
+        or pipeline_name == "inpatient_charges_2013"
+        or pipeline_name == "inpatient_charges_2014"
+        or pipeline_name == "inpatient_charges_2015"
+        or pipeline_name == "outpatient_charges_2012"
+        or pipeline_name == "outpatient_charges_2013"
+        or pipeline_name == "outpatient_charges_2014"
+        or pipeline_name == "outpatient_charges_2011"
+    ):
+
+        with ZipFile(source_file) as zipped_files:
+
+            file_list = zipped_files.namelist()
+            csv_file = fnmatch.filter(file_list, "*.csv")
+            data = zipped_files.open(*csv_file)
+            df = pd.read_csv(data)
+
+    else:
+
+        df = pd.read_csv(str(source_file))
+
+    logging.info(f"Transformation Process Starting.. {source_file}")
+
+    logging.info(f"Transform: Renaming Headers.. {source_file}")
```

Comment:
We don't need these log statements because your function is already self-explanatory, i.e. `rename_headers(...)`.

Kindly review the rest of the file for the other log statements.

Reply:
Removed.

Reply:
Removed.